### PR TITLE
Create custom-csv-settings.md

### PR DESCRIPTION
**Pull request by dalholm**
I think should be in the docs. 👍

### CODE REVIEW

**Comment by patrickbrouwers** [comment]
Thanks @dalholm , can you also add the page to the sidebar: https://github.com/Maatwebsite/laravel-excel-docs/blob/master/.vuepress/3.1.js#L37

File: 3.1/imports/custom-csv-settings.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+By default Laravel Excel uses default standards for CSV reading. You can change this by adding the WithCustomCsvSettings interface.
+
+```php
+namespace App\Imports;
+
+use Maatwebsite\Excel\Concerns\ToModel;
+use Maatwebsite\Excel\Concerns\WithCustomCsvSettings;
+
+class UsersImport extends DefaultValueBinder implements ToModel, WithCustomCsvSettings
```
Comment (patrickbrouwers) [inline 3.1/imports/custom-csv-settings.md]: `extends  DefaultValueBinder` can be removed

File: 3.1/imports/custom-csv-settings.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Custom CSV Settings
+
+By default Laravel Excel uses default standards for CSV reading. You can change this by adding the WithCustomCsvSettings interface.
```
Comment (patrickbrouwers) [inline 3.1/imports/custom-csv-settings.md]: it uses the defaults from the config (config/excel.php)

**Comment by dalholm** [comment]
> Thanks @dalholm , can you also add the page to the sidebar: https://github.com/Maatwebsite/laravel-excel-docs/blob/master/.vuepress/3.1.js#L37

This is added now.

**Comment by patrickbrouwers** [comment]
Looks good, thanks!

**Pull request merged**